.gitattributes

```diff
@@ -0,0 +1,6 @@
+# Dependency lockfile generated by `flutter pub`
+pubspec.lock linguist-generated=true
+# Metadata generated by flutter
+.metadata linguist-generated=true
+# Generated files by the theme_tailor library
+*.tailor.dart linguist-generated=true
```

Comment:
All files ending in `.tailor.dart` are automatically generated by the `theme_taylor` dev dependency. They have a `// GENERATED CODE - DO NOT MODIFY BY HAND` at the top.

Because they are autogenerated, I felt like they don't add much value in the diff, and [by marking them like this they are collapsed by default on GitHub](https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github). If you don't want it I can remove the the attributes again.